editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -509,6 +527,9 @@ function getGridMoveType(params: {
   // For absolute move, just use rearrange.
   // TODO: maybe worth reconsidering in the future?
```

Comment:
we can maybe remove this comment now? or is it still relevant for when it's absolute with grid positioning?

Reply:
good point! removing